ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/workloadCveOverviewPage.test.ts

```diff
@@ -51,6 +52,19 @@ describe('Workload CVE overview page tests', () => {
             'aria-pressed',
             'true'
         );
+        cy.getTelemetryEvents().should((telemetryEvents) => {
+            // TODO - This is not working as expected. We should only have one page view event, but the page view event is being emitted twice.
```

Comment:
If `<MainPage>` rerenders, we will get duplicate page view events. This is a problem unrelated to the regression and the fix in this PR, but is something we should deal with eventually. The re-rendering does not happen 100% of the time, but can be reproduced reliably.